Index: src/main/java/com/example/Project2/chatController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/Project2/chatController.java b/src/main/java/com/example/Project2/chatController.java
--- a/src/main/java/com/example/Project2/chatController.java	
+++ b/src/main/java/com/example/Project2/chatController.java	
@@ -14,8 +14,9 @@
 import javafx.stage.StageStyle;
 
 import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.sql.*;
 import java.util.ArrayList;
 
 public class chatController extends onderwerp{
@@ -98,10 +99,9 @@
                 Instellingen.setText("Settings");
                 Profiel.setText("Profile");
 
+            }}}
 
-            }
-        }
-    }
+
 
     public void VonderwerpOnAction(ActionEvent event){
         onderwerp1 = null;
Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/pom.xml b/pom.xml
--- a/pom.xml	
+++ b/pom.xml	
@@ -102,6 +102,12 @@
             <version>RELEASE</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>RELEASE</version>
+            <scope>compile</scope>
+        </dependency>
     </dependencies>
 
     <build>
Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	
+++ b/src/main/java/module-info.java	
@@ -4,6 +4,7 @@
     requires javafx.fxml;
     requires java.sql;
     requires mysql.connector.java;
+    requires org.junit.jupiter.api;
 
 
     opens com.example.Project2 to javafx.fxml;
